server/src/main/scala/services/workloadcalculator/WorkloadCalculator.scala

```diff
@@ -32,9 +32,9 @@ object PaxLoadCalculator {
     queueWithPaxloads
   }
 
-  def voyagePaxSplitsFromApiFlight(splitsRatioProvider: (ApiFlight) => List[SplitRatio])(flight: ApiFlight): IndexedSeq[(Long, PaxTypeAndQueueCount)] = {
+  def voyagePaxSplitsFromApiFlight(splitsRatioProvider: (ApiFlight) => Option[List[SplitRatio]])(flight: ApiFlight): IndexedSeq[(Long, PaxTypeAndQueueCount)] = {
     val timesMin = new DateTime(flight.SchDT, DateTimeZone.UTC).getMillis
-    val splits = splitsRatioProvider(flight)
+    val splits = splitsRatioProvider(flight).get
```

Comment:
Throw exception in this case?

It would mean there was no set of default splits configured